crates/cargo-rm/tests/testsuite/cargo_rm/invalid_arg/stderr.log

```diff
@@ -3,6 +3,6 @@ error: Found argument '--flag' which wasn't expected, or isn't valid in this con
 	If you tried to supply `--flag` as a value rather than a flag, use `-- --flag`
 
 USAGE:
-    cargo rm <DEP_ID>...
+    cargo-rm rm <DEP_ID>...
```

Comment:
You can set `#[clap(bin_name = "cargo")]` on  the builder to avoid this though.  While it is in the part of the code we'll drop when we port to cargo, it reduces the test churn when doing so.

Reply:
Ah, didn't know that, thanks!